src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/BasicTagHelpers_Prefixed_DesignTime.codegen.html

```diff
@@ -0,0 +1,10 @@
+~~~~~~~~~~~~~~~~ ~~~~~
+~~~~~~~~~~~~~ ~~~ ~~~~~~~~~~~~~
```

Comment:
I assume getting rid of the tildes is something for a later PR?

Reply:
Yes, it breaks formatting when we do that, so wanted to keep the review smaller